index.html

```diff
@@ -9,6 +9,12 @@
       name="og:description"
       content="The homepage for our lovely little community."
     />
+    <meta name="description" content="We’re a group of members committed to providing a safe, and comfortable space for anyone to try their hand at coding, and have some fun along the way." />
+    <meta property="og:image" content="https://user-images.githubusercontent.com/51391473/144103613-229d1856-7815-4c14-b43c-229646bc66fb.png" />
+    <meta property="og:url" content="https://community.nhcarrigan.com/" />
+    <meta property="og:type" content="website" />
+    <meta property="og:locale" content="en_US" />
+    <meta property="og:description" content="We’re a group of members committed to providing a safe, and comfortable space for anyone to try their hand at coding, and have some fun along the way." />
```

Comment:
```suggestion
    <meta property="og:description" content="We’re a group of members committed to providing a safe, and comfortable space for anyone to try their hand at coding, and have some fun along the way." />
    <meta property="twitter:card" content="summary_large_image" />
    <meta property="twitter:creator" content="@nhcarrigan" />
```
Let's enable the twitter embed too 🙂